.. _set_exception:

yr.object_ref.ObjectRef.set_exception
--------------------------------------

.. py:method:: ObjectRef.set_exception(e)

    设置异常。